source/core/parameters_lagrangian.cc

```diff
@@ -135,19 +135,23 @@ namespace Parameters
     {
       prm.enter_subsection("lagrangian physical properties");
       {
-        prm.declare_entry("gx",
-                          "0.",
-                          Patterns::Double(),
-                          "Gravitational acceleration in x direction");
-        prm.declare_entry("gy",
-                          "0.",
-                          Patterns::Double(),
-                          "Gravitational acceleration in y direction");
-        prm.declare_entry("gz",
-                          "0.",
-                          Patterns::Double(),
-                          "Gravitational acceleration in z direction");
-
+        // Parameter <g> is a list of values, its deprecated version are
+        // individual parameters <gx>, <gy> and <gz>
```

Comment:
```suggestion
        // Parameter <g> is a list of values, its deprecated version is
        // individual parameters <gx>, <gy> and <gz>
```

Reply:
For me it is one version that contains 3 parameters